src/index.ts

```diff
@@ -100,9 +107,19 @@ program
     .description('Remove trusted root identity by id')
     .action(removeTrustedAuthority);
 
+//create verifiable credential
+program
+    .command('create-vc')
+    .requiredOption('-i, --identityFile <path-to-identity-file>')
```

Comment:
you changed for the commands search-identity and find-identity the option description for -identityFile from <path-to-identity-file> to  <Identity file>. 
Here and and at the rest of the commands you let <path-to-identity-file>?

Reply:
I will add "path-to-identity-file" to all